Task7/src/index.js

```diff
@@ -0,0 +1,183 @@
+        let elements = document.querySelectorAll(selector);
+        for(let i = 0; i < elements.length; i++ ) {
+            elements[i].classList.remove(className);
+        }
+    };
+
+    append(strHtml){
+        jQuery.append(this._string, strHtml);
+        return this;
+    }
+
+    static append(selector, strHtml){
+        let elements = document.querySelectorAll(selector);
+        for(let i = 0; i < elements.length; i++ ) {
+            elements[i].insertAdjacentHTML('afterbegin', strHtml);
+        }        
+    };
+
+    remove(){
+        jQuery.remove(this._string);
+        return this;
+    }
+
+    static remove(selector){
+        let elements = document.querySelectorAll(selector);
+        for(let i = 0; i < elements.length; i++ ){
+            elements[i].parentNode.removeChild(elements[i]);
+        }        
+    };
+
+    text(){
+        return jQuery.text(this._string);
+    }
+
+    static text(node){
+        let element = document.querySelector(node);
+        return element.innerText;
+    };
+
+    attr(attributeName, value){
+        return jQuery.attr(this._string, attributeName, value);
+    };
+
+    //Get the value of an attribute for the first element in the set of matched elements
+    //value != null set one or more attributes for every matched element.
+    static attr(selector, attributeName, value){
+        let element = document.querySelector(selector);
+        if (value){
+            element.setAtttibute(attributeName, value);
+        }
+        else{
+            return element.getAttribute(attributeName);            
+        }
+    };
+
+    children(selector){
+        return new jQuery( jQuery.children(this._string, selector));
+    }
+
+    //Get the children of each element in the set of matched elements, optionally filtered by a selector.
+    static children(node, selector){
+        let element = document.querySelector(node);
+        let result = [];
+        if (!selector)
+            return element.childNodes;
+
+        for(let i = 0; i < element.childNodes.length; i++ ){            
+            if (element.childNodes[i].nodeType == 1 && 
+                (element.childNodes[i].matches == "undefined"
+                ? element.childNodes[i].matches(selector)
+                :element.childNodes[i].msMatchesSelector(selector)
+                )){
+                result.push(element.childNodes[i]);
+            }
+        }
+        return result;
+    };
+
+    empty(){
+        jQuery.empty(this._string);
+        return;
+    }
+
+    //Remove all child nodes of the set of matched elements from the DOM.
+    static empty(node){
+        let element = document.querySelector(node);
+        
+        while (element.firstChild){
+            let child = element.firstChild;
+            child.parentNode.removeChild( child);   
+        }        
+    };
+
+    css(property){
+        jQuery.css(this._string, property);
+        return this;
+    }
+
+    static css(node, property){
```

Comment:
I am a bit confused how it works. You can pass `property` but you can't pass `value`.

See https://api.jquery.com/css/

Reply:
Corrected